### PR TITLE
ci: add issues to Optimize project

### PR DESCRIPTION
**Pull request by misiekhardcore**
## Description

This PR adds auto labeling and project assigning functionality for Optimize

## Related issues

closes https://github.com/camunda/camunda-optimize/issues/13886

### CODE REVIEW

**Review by buccarel — APPROVED** [review]

**Review by RomanJRW — APPROVED** [review]
Changes are fine for me. However, please consider:
- Target branch: This could probably already target `main`, there is no dependency on the "merge in" PR afaict
- It might be that @cmur2 also wants to review this. While I can't see an objection, it is not a bad thing to give him visibility on the changes that are being made in the mono repo workflows. Firstly, because it's his domain, and secondly because it can help him in making a more informed review on the merge-in PR. Note that you could also agree that he does this after its merged to the merge-in branch, if that is the approach you indeed take

**Comment by cmur2** [comment]
I agree with Josh's comment and thanks for looping me in! PR looks good from my side (eventually we should deduplicate those workflows since the logic is always the same, except parameters).

**Comment by misiekhardcore** [comment]
@cmur2 I noticed that when I wan working on this that there is a lot of duplication. If you create/have an issue to abstract this common logic into an action, I can work on that

**Comment by misiekhardcore** [comment]
@buccarel how do you want this to be merged? to `main` or to `merge-optimize-in`?

File: .github/workflows/optimize-labeled.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                issue: issue(number: $issue) {
+                  projectsV2 {
+                    totalCount
+                  }
+                }
+              }
+            }
+          variables: |
+            owner: "camunda"
+            repo: "zeebe"
```
Comment (cmur2) [inline .github/workflows/optimize-labeled.yml:28]: This is now `camunda´ (technically works with `zeebe` but better to update)

**Comment by cmur2** [comment]
@misiekhardcore the refactoring ticket is https://github.com/camunda/camunda/issues/20768

File: .github/workflows/optimize-labeled.yml (hunk 1 above)
Comment (misiekhardcore) [reply .github/workflows/optimize-labeled.yml]: renamed

**Pull request merged**